clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -645,8 +645,16 @@ void CodeGenModule::Release() {
     SPIRVerMD->addOperand(llvm::MDNode::get(Ctx, SPIRVerElts));
     // We are trying to look like OpenCL C++ for SPIR-V translator.
     // 4 - OpenCL_CPP, 100000 - OpenCL C++ version 1.0
+    // 6 - ESIMD, if any kernel or function is an explicit SIMD one
+    int Lang = 4;
```

Comment:
This whole initialization should be some sort of llvm algorithm/std::algorithm here.

Reply:
ok

Reply:
Seems that this is not resolved yet.

Reply:
Right, will fix